python/kiss_icp/tools/visualizer.py

```diff
@@ -215,15 +234,26 @@ def _update_geometries(self, source, keypoints, target, pose):
 
         # Target Map
         if self.render_map:
-            target = copy.deepcopy(target)
+            target = target_map.point_cloud()
             self.target.points = self.o3d.utility.Vector3dVector(target)
             if self.global_view:
-                self.target.paint_uniform_color(GRAY)
+                #  self.target.paint_uniform_color(GRAY)
+                pass
             else:
                 self.target.transform(np.linalg.inv(pose))
         else:
             self.target.points = self.o3d.utility.Vector3dVector()
 
+        # VoxelHashMap
+        if self.render_voxel_grid and self.global_view:
+            self.voxel_map.clear()
+            self.voxel_map.voxel_size = self.voxel_size
+            voxels = target_map.get_voxels()
+            for voxel in voxels:
+                self.voxel_map.add_voxel(self.o3d.geometry.Voxel(voxel, LIGHT_GRAY))
```

Comment:
This will require `open3d>v0.18.0`, see https://github.com/isl-org/Open3D/pull/6023, which we have to change in the `pyproject.toml`.

But anyway obsolete if we do the voxel visualization with the new kissualizer :)

Reply:
Yes yes, this was a draft proposal with the idea in mind that will be obsolete the moment I open the PR. Thanks for the clarification